test/rantly_generator_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+require 'rantly/minitest_extensions'
+
+describe Rantly do
+  before do
+    Rantly.gen.reset
+  end
+
+  describe '#range' do
+    it 'returns an integer within range when given two integers' do
+      property_of {
```

Comment:
[nitpick] I know you are only moving this code, but I would put this two properties in different `it`, to have [single expectation test](http://www.betterspecs.org) (at least when it is easy to achieve as here). Also, the first one is not really testing that it returns an integer within range when given two integers, but that it works when the two integers are the same.

Reply:
Split this spec into two.